Dockerfile

```diff
@@ -72,6 +73,11 @@ RUN echo "👋 Welcome to Codespaces! You are on a custom image defined in your
     && echo "🔍 To explore VS Code to its fullest, search using the Command Palette (Cmd/Ctrl + Shift + P)\n" >> /usr/local/etc/vscode-dev-containers/first-run-notice.txt \
     && echo "👋 Welcome to the Kind-in-Docker image\n" >> /usr/local/etc/vscode-dev-containers/first-run-notice.txt
 
+RUN apt-get update -y
```

Comment:
same for 76-79 and 91-94, 120-123, 172-175 combine in a single RUN layer

Reply:
we do that intentionally so that the layers pull in parallel and startup is faster. There's a comment on line 16